huaweicloud/services/fgs/resource_huaweicloud_fgs_trigger.go

```diff
@@ -271,6 +271,16 @@ func kafkaSchemaResource() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"kafka_user": {
```

Comment:
the structure already names `kafka`, the sub-parameters are not recommand start with the `kafka_`.